src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -22,54 +22,44 @@
 // ***********************************************************************
 
 using System;
+using System.Xml;
+
+#if PORTABLE || SILVERLIGHT
+using System.Xml.Linq;
+#endif
 
 namespace NUnit.Framework.Interfaces
 {
     /// <summary>
-    /// XmlNode represents a single node in the XML representation
+    /// TNode represents a single node in the XML representation
     /// of a Test or TestResult. It replaces System.Xml.XmlNode and
-    /// provides a minimal set of methods for operating on the XML 
-    /// in a platform-independent manner.
+    /// System.Xml.Linq.XElement, providing a minimal set of methods 
+    /// for operating on the XML in a platform-independent manner.
     /// </summary>
-    public class XmlNode
+    public class TNode
```

Comment:
Why TNode for the name? I am probably being overly picky, but when I read `TNode` in `IXmlNodeBuilder` I expected it to be a generic method.